hibernate-spatial/src/main/java/org/hibernate/spatial/predicate/JTSSpatialPredicates.java

```diff
@@ -63,9 +65,7 @@ public static Predicate eq(
 	public static Predicate eq(
 			CriteriaBuilder criteriaBuilder, Expression<? extends Geometry> geometry1,
 			Geometry geometry2) {
-		return eq( criteriaBuilder, geometry1,
-				criteriaBuilder.literal( geometry2 )
-		);
+		return eq( criteriaBuilder, geometry1, criteriaBuilder.literal( geometry2 ) );
```

Comment:
Please note that we have a special function `value` now that has a configuration to decide if this should be rendered as literal or parameter into the SQL, so please change all uses in this class.

Reply:
I wasn't aware of that. I will make the requested changes in a separate commit